src/PublicApiAnalyzers/PublicApiAnalyzers.Help.md

```diff
@@ -35,15 +35,11 @@ For example when using the [`#if` preprocessor directive](https://docs.microsoft
 
 To correctly model the API differences between target frameworks (or any other property), use multiple instances of the `PublicAPI.*.txt` files.
 
-For example, if you target both `net4.8` and `netcoreapp3.0` target frameworks, and APIs differ between each, then you would have the following:
+For example, if you target both `net48` and `netcoreapp3.0` target frameworks, and APIs differ between each, then you would have the following:
```

Comment:
I prefer being explicit with the `.` as it is less confusing for people who are new to these monikers. Why not `netcoreapp30` too?

Given the code sample no longer makes explicit mention of any moniker, I'd shorten this sentence to:

```suggestion
If you have multiple target frameworks and APIs differ between them, use the following in your project file:
```

Reply:
> Why not netcoreapp30 too?

I was under the impression that this doesn't work. I think I tried it with netcoreapp1.0 :D